spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerAutoConfiguration.java

```diff
@@ -96,31 +99,22 @@ public AxonServerConnectionManager platformConnectionManager(AxonServerConfigura
 
     @Bean( destroyMethod = "disconnect")
     @Primary
-    @ConditionalOnMissingBean(CommandBus.class)
-    public AxonServerCommandBus commandBus(TransactionManager txManager,
+    @ConditionalOnMissingQualifiedBean(qualifier = "!unqualified", beanClass = CommandBus.class)
+    public AxonServerCommandBus axonServerCommandBus(TransactionManager txManager,
```

Comment:
Nit: indenting of this function is a little off.

Reply:
Fixed